<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFSPIMLightsheetEvaluationEditor</class>
 <widget class="QWidget" name="QFSPIMLightsheetEvaluationEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Preview &amp;&amp; Single Fits</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0">
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>color channel:</string>
            </property>
            <property name="buddy">
             <cstring>cmbChannel</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbChannel"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>stack position:</string>
            </property>
            <property name="buddy">
             <cstring>spinStackPos</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="horizontalScrollBar">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinStackPos"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="JKQtPlotter" name="pltImage" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="JKQtPlotter" name="pltFit" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labFitResults">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Stack Fit Results</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,1,0,0,1" columnstretch="1,1">
       <item row="5" column="0">
        <widget class="JKQtPlotter" name="pltParam" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>beam position:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="JKQtPlotter" name="pltBeamPos" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="JKQtPlotter" name="pltBeamPosDifference" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>lightsheet pos. range:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinRangeMin">
            <property name="suffix">
             <string> µm</string>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinRangeMax">
            <property name="suffix">
             <string> µm</string>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>width range:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinWidthRangeMin">
            <property name="suffix">
             <string> µm</string>
            </property>
            <property name="maximum">
             <double>100000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinWidthRangeMax">
            <property name="suffix">
             <string> µm</string>
            </property>
            <property name="maximum">
             <double>100000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>beam width:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="JKQtPlotter" name="pltBeamWidth" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>parameter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbParameter">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkErrorsParam">
           <property name="text">
            <string>show errors</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>beam position difference:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkErrorsBeamPos">
           <property name="text">
            <string>show errors</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Δz&lt;/span&gt; = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinDeltaZ</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="spinDeltaZ">
         <property name="suffix">
          <string> nm</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Δx&lt;/span&gt; = &lt;span style=&quot; font-style:italic;&quot;&gt;Δy =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="buddy">
          <cstring>spinDeltaX</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="spinDeltaX">
         <property name="suffix">
          <string> nm</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
         <property name="value">
          <double>400.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>orientation:</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrientation</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cmbOrientation">
         <item>
          <property name="text">
           <string>fit rows</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fit columns</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="chkUseMask">
       <property name="text">
        <string>use mask (if available)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnEvaluateCurrent">
       <property name="text">
        <string>Evaluate &amp;Current</string>
       </property>
       <property name="icon">
        <iconset resource="qfspimlightsheetevaluation.qrc">
         <normaloff>:/spim_lightsheet_eval/qfspimlightsheetevaluation_evaluate.png</normaloff>:/spim_lightsheet_eval/qfspimlightsheetevaluation_evaluate.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveReport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save &amp;Report</string>
       </property>
       <property name="icon">
        <iconset resource="qfspimlightsheetevaluation.qrc">
         <normaloff>:/spim_lightsheet_eval/qfeval_savereport.png</normaloff>:/spim_lightsheet_eval/qfeval_savereport.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrintReport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Print Report</string>
       </property>
       <property name="icon">
        <iconset resource="qfspimlightsheetevaluation.qrc">
         <normaloff>:/spim_lightsheet_eval/qfeval_printreport.png</normaloff>:/spim_lightsheet_eval/qfeval_printreport.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;stack:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>cmbStack</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbStack"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFFitFunctionComboBox" name="cmbModel"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>     algorithm:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFFitAlgorithmComboBox" name="cmbAlgorithm"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQtPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFFitFunctionComboBox</class>
   <extends>QComboBox</extends>
   <header>qffitfunctioncombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFFitAlgorithmComboBox</class>
   <extends>QComboBox</extends>
   <header>qffitalgorithmcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qfspimlightsheetevaluation.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinStackPos</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinStackPos</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>displayPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbChannel</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>displayPreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbStack</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>updateStack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>1009</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinStackPos</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRangeMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>updateFitResultRanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRangeMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>updateFitResultRanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinWidthRangeMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>updateFitResultRanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinWidthRangeMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFSPIMLightsheetEvaluationEditor</receiver>
   <slot>updateFitResultRanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayPreview()</slot>
  <slot>updateStack()</slot>
  <slot>updateFitResultRanges()</slot>
 </slots>
</ui>
